datahub-web-react/src/app/entity/shared/entity/EntityActions.tsx

```diff
@@ -117,6 +117,7 @@ function EntityActions(props: Props) {
                     fixedEntityTypes={Array.from(
                         entityRegistry.getTypesWithSupportedCapabilities(EntityCapabilityType.GLOSSARY_TERMS),
                     )}
+                    isBatchAddGlossaryTermModalVisible={isBatchAddGlossaryTermModalVisible}
```

Comment:
what about any of the other batch actions? like setting domains and other bulk situations this will only fix the css issue if it's glossary terms.